### DISCUSSION TITLE
Authorization using Custom Directives and Netflix DGS

### DISCUSSION THREAD

**Discussion by ankit-joinwal** [Netflix/dgs-framework#422] [Show and tell]
Hello Awesome Community,

I have created a working example of how to use Custom directives for doing authorization checks in GraphQL using Netflix DGS.

https://github.com/ankit-joinwal/netflix-dgs-authorisation-directives

I want to know if my example project can be included in DGS examples repository.

CC @paulbakker @srinivasankavitha

**Comment by ankit-joinwal** [comment]
Coming soon - A blog post that explains the design and implementation.

**Reply by srinivasankavitha** [reply]
Thanks for putting this together! This is very useful.

**Reply by srinivasankavitha** [reply]
We will set up a section in our docs for community contributed examples that folks can contribute to and link to your example there.

**Reply by srinivasankavitha** [reply]
https://github.com/Netflix/dgs/pull/47
Please let me know if you have any objections.

**Reply by ankit-joinwal** [reply]
Thank you for accepting the example and providing a way forward to contribute.
